WcaOnRails/app/models/competition.rb

```diff
@@ -809,6 +805,24 @@ def self.search(query, params: {}, managed_by_user: nil)
     competitions.includes(:delegates, :organizers).order(start_date: :desc)
   end
 
+  # See https://github.com/thewca/worldcubeassociation.org/wiki/wcif
+  def to_wcif
+    managers = self.managers
+    persons_wcif = registrations.map do |r|
+      managers.delete(r.user)
+      r.user.to_wcif(self, r.to_wcif)
+    end
+    # Note: unregistered managers may generate N+1 queries on their personal bests,
+    # but that's fine because there are very few of them!
+    persons_wcif += managers.map { |m| m.to_wcif(self) }
```

Comment:
I believe all of the managers specific code can get removed, because we're planning on doing #1568, right? Or do you plan to remove this code as part of #1568?